src/i18n/fr_FR.ts

```diff
@@ -0,0 +1,821 @@
+            close: 'Fermer ce message et recharger le jeu (recommandé)',
+            abort: 'Fermer ce message sans recharger le jeu',
+        },
+    },
+    globalSettings: {
+        name: 'Paramètres généraux',
+        labelInMenu: {
+            title: 'Titre au lieu d\'une icone dans le menu',
+            description:
+                'Un simple titre dans la barre de navigation au lieu du logo LSSM',
+        },
+    },
+    vehicles: [
+        {
+            caption: 'FPT',
+            color: '#cc0000',
+            coins: 25,
+            credits: 5_000,
+            minPersonnel: 4,
+            maxPersonnel: 6,
+            wtank: 0,
+        },
+        {
+            caption: 'FPTL',
+            color: '#bb0000',
+            coins: 25,
+            credits: 5_000,
+            minPersonnel: 4,
+            maxPersonnel: 6,
+            wtank: 0,
+        },
+        {
+            caption: 'EPA',
+            color: '#d92626',
+            coins: 30,
+            credits: 10_000,
+            minPersonnel: 1,
+            maxPersonnel: 2,
+            wtank: 0,
+            special: 'Nécessaire dès lors que vous avez construit 3 bases de pompiers',
+        },
+        {
+            caption: 'VLCG',
+            color: '#d02525',
+            coins: 25,
+            credits: 10_000,
+            minPersonnel: 1,
+            maxPersonnel: 1,
+            wtank: 0,
+            special: 'Nécessaire dès lors que vous avez construit 6 bases de pompiers',
+        },
+        {
+            caption: 'VSR',
+            color: '#ad1f1f',
+            coins: 25,
+            credits: 12_180,
+            minPersonnel: 2,
+            maxPersonnel: 3,
+            wtank: 0,
+            special: 'Nécessaire dès lors que vous avez construit 4 bases de pompiers',
+        },
+        {
+            caption: 'UMH',
+            color: '#9c1c1c',
+            coins: 25,
+            credits: 5_000,
+            minPersonnel: 3,
+            maxPersonnel: 3,
+            wtank: 0,
+        },
+        {
+            caption: 'CCGC',
+            color: '#aa0000',
+            coins: 25,
+            credits: 17_300,
+            minPersonnel: 1,
+            maxPersonnel: 2,
+            wtank: 0,
+            special: 'Nécessaire dès lors que vous avez construit 7 bases de pompiers',
+        },
+        {
+            caption: 'VIRT',
+            color: '#770000',
+            coins: 25,
+            credits: 19_200,
+            minPersonnel: 1,
+            maxPersonnel: 4,
+            wtank: 0,
+            special: 'Nécessaire dès lors que vous avez construit 7 bases de pompiers',
+            schooling: 'Véhicules risques chimiques',

```

Comment:
add this Centre de secours - Véhicules risques chimiques . what we need here is the training as it is in the course window, as with the alliance.